src/plugins/anti_caching_plugin.cpp

```diff
@@ -0,0 +1,47 @@
+#include "anti_caching_plugin.hpp"
+
+#include <iostream>
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+const std::string AntiCachingPlugin::description() const
+{
+  return "AntiCaching Plugin";
+}
+
+void AntiCachingPlugin::start()
+{
+  std::cout << "AntiCaching Plugin starting up.\n";
+  _evaluate_statistics_thread =
+    std::make_unique<PausableLoopThread>(REFRESH_STATISTICS_INTERVAL, [&](size_t) { _evaluate_statistics(); });
+
+}
+
+void AntiCachingPlugin::stop()
+{
+  std::cout << "AntiCaching Plugin stopping.\n";
+
+  _evaluate_statistics_thread.reset();
+}
+
+void AntiCachingPlugin::_evaluate_statistics()
+{
+  std::cout << "Evaluating statistics\n";
+  // warum greifen wir die Statistiken nicht hier ab und setzen sie dann in den Segmenten zurück?
```

Comment:
That would mean that only one plugin (or other component) could use the statistics, wouldn't it?

Reply:
Currently I am copying the the counters and reset them afterwards. That might not be the best idea :-)